### DISCUSSION TITLE
How to get the bounding box of the current screen?

### DISCUSSION THREAD

**Discussion by gcsaba2** [liodali/osm_flutter#195] [Q&A] [answered]
Assuming I have a map as in the screenshot below, how to I get the bounding box of the currently visible screen?

I am able to get my current GPS location, but the map does not necessarily need to be there at the moment. I would like to get the currently visible location in order to load data for the visible map only.

I didn't find any methods to do this. Is there a way?

![example map](https://user-images.githubusercontent.com/53976741/145465110-be13ed20-7709-48ff-9a03-f07db3a4f9d0.png)

**Comment by liodali** [comment, marked as answer]
in 0.27 i will add a method that will gives you bounding box 
we have already PR that  implement region changed and i added to it bounding box properties
it will be available soon

**Reply by gcsaba2** [reply]
Thank you, this is excellent news!
